src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks.mdx

```diff
@@ -21,13 +21,11 @@ The easiest way to see what your user management permissions are is to go to the
 
 Some user management requirements and restrictions:
 
-* This doc is about managing users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models). To learn more, see [User models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models).
-* To avoid configuration conflicts, try to ensure that only one person in your organization is managing users at a time. User management by more than one user at the same time may result in errors. 
+* This doc is about managing users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models). 
 * Most user management capabilities require the [**Authentication domain manager** role](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#standard-roles) and some require the [**Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#standard-roles). For details, see [Standard roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles).
-* Pricing edition requirements:
-  * To manage user groups, roles, and access grants: [Pro or Enterprise](https://newrelic.com/pricing) edition is required.
-  * To [import users from an identity provider](/docs/accounts/accounts-billing/new-relic-one-user-management/configure-authentication-domains-sso#source-users): [Enterprise](https://newrelic.com/pricing) is required.
-* A New Relic user can have a maximum of either three concurrent active sessions, or three unique IP addresses in use at any given time.
+* To manage groups, roles, and access grants, [Pro or Enterprise](https://newrelic.com/pricing) edition is required.
+* A New Relic user is prohibited from sharing their login with other people. A New Relic user can have a maximum of either three concurrent active sessions, or three unique IP addresses in use at any given time.
+* To avoid configuration conflicts, try to ensure that only one person in your organization is managing users at a time. User management by more than one user at the same time may result in errors. 
```

Comment:
`User management by more than one user at the same time may result in errors. ` is a bit hard to parse, I think because `User management by more than one user` is the object of the sentence and that's a bit of a mouthful. I actually think that you can delete it entirely—the first sentence already explains why.

If you want to keep it, I'd suggest expanding it more to instead explain _why_ not to do this. For example, the bullet could become something like:

```Try to ensure that only one person in your organization is managing users at a time. Otherwise, you could create configuration conflicts and run into errors or unexpected behavior.```